changelog.txt

```diff
@@ -31,6 +31,7 @@ Added
  - Enable deleting of `JSONDict` keys through the attribute interface, e.g., `del job.doc.foo`.
  - The `Job.init()` method returns the job to allow one-line job creation and initialization.
  - Add HTML representation of instances of `Project` and `JobsCursor` targeted at Jupyter Notebooks (requires pandas).
+ - Code coverage is now uploaded to https://codecov.io/gh/glotzerlab/signac/
```

Comment:
I don't think that's something we should announce in the changelog.

Reply:
Sorry, I'll retract this commit. Just checking the boxes on the PR form. ;)

Reply:
Sorry, I'll retract this commit. Just checking the boxes on the PR form. ;)